documentation/RFC/RFC0001-Aure-Profiles.md

```diff
@@ -0,0 +1,61 @@
+---
+RFC: RFC0001
+Author: Azure PowerShell team
+Status:  Draft
+SupercededBy: n/a
+Version: 1.0
+---
+
+# Enable Azure Profiles with Azure PowerShell
+
+Azure comprises several clouds (Public, Azure Stack, Government), each of which comprises several services (APIs) that have different versions.
+
+Writting an application or a script that is compatible with several cloud can quickly become a very complex task as the developer has to identify for each operation to be performed which API version is compatible on each cloud.
+As Azure evolves, existing services will expose new resources or properties and potentially change how to interact with those, subject to breaking changes. We are adding the support of Azure profiles in Azure PowerShell. An Azure profiles is a definition of resources providers and associated API Version.
```

Comment:
We are adding support for Azure profiles to help manage this complexity.  Profiles provide a simple version across the entire set of azure services.  When writing a script, simply select a profile for all services, rather than choosing a specific api-version for each service.  This allows scripts to have a consistent api surface that they can be sure will work across the targeted clouds.